src/Evolution/Systems/ForceFree/Tags.hpp

```diff
@@ -0,0 +1,190 @@
+
+namespace ForceFree {
+
+/*!
+ * \brief Tags for the GRFFE system with divergence cleaning.
+ */
+namespace Tags {
+
+/*!
+ * \brief The electric field \f$E^i\f$.
+ */
+struct ElectricField : db::SimpleTag {
+  using type = tnsr::I<DataVector, 3>;
+};
+
+/*!
+ * \brief The magnetic field \f$B^i\f$.
+ */
+struct MagneticField : db::SimpleTag {
+  using type = tnsr::I<DataVector, 3>;
+};
+
+/*!
+ * \brief The electric charge density \f$q\equiv-n_\mu J^\mu\f$ where
+ * \f$n^\mu\f$ is the normal to hypersurface and \f$J^\mu\f$ is the 4-current.
+ */
+struct ChargeDensity : db::SimpleTag {
+  using type = Scalar<DataVector>;
+};
+
+/*!
+ * \brief The spatial electric current density \f$J^i\f$.
+ */
+struct SpatialCurrentDensity : db::SimpleTag {
+  using type = tnsr::I<DataVector, 3>;
+};
+
+/*!
+ * \brief The divergence cleaning scalar field \f$\psi\f$ coupled to the
+ * electric field.
+ */
+struct ElectricDivergenceCleaningField : db::SimpleTag {
+  using type = Scalar<DataVector>;
+};
+
+/*!
+ * \brief The divergence cleaning scalar field \f$\phi\f$ coupled to the
+ * magnetic field.
+ */
+struct MagneticDivergenceCleaningField : db::SimpleTag {
+  using type = Scalar<DataVector>;
+};
+
+/*!
+ * \brief The densitized electric field \f$\tilde{E}^i = \sqrt{\gamma}E^i\f$.
+ */
+struct TildeE : db::SimpleTag {
+  using type = tnsr::I<DataVector, 3>;
+};
+
+/*!
+ * \brief The densitized magnetic field \f$\tilde{B}^i = \sqrt{\gamma}B^i\f$.
+ */
+struct TildeB : db::SimpleTag {
+  using type = tnsr::I<DataVector, 3>;
+};
+
+/*!
+ * \brief The densitized divergence cleaning field \f$\tilde{\psi} =
+ * \sqrt{\gamma}\psi\f$ associated with the electric field.
+ */
+struct TildePsi : db::SimpleTag {
+  using type = Scalar<DataVector>;
+};
+
+/*!
+ * \brief The densitized divergence cleaning field \f$\tilde{\phi} =
+ * \sqrt{\gamma}\phi\f$ associated with the magnetic field.
+ */
+struct TildePhi : db::SimpleTag {
+  using type = Scalar<DataVector>;
+};
+
+/*!
+ * \brief The densitized electric charge density \f$\tilde{q} =
+ * \sqrt{\gamma}q\f$.
+ */
+struct TildeQ : db::SimpleTag {
+  using type = Scalar<DataVector>;
+};
+
+/*!
+ * \brief The largest characteristic speed
+ */
+struct LargestCharacteristicSpeed : db::SimpleTag {
+  using type = double;
+};
+}  // namespace Tags
+
+namespace OptionTags {
+/*!
+ * \ingroup OptionGroupsGroup
+ * \brief Groups option tags related to the GRFFE evolution system.
+ */
+struct ForceFreeGroup {
+  static std::string name() { return "ForceFree"; }
+  static constexpr Options::String help{
+      "Options for the GRFFE evolution system"};
+  using group = evolution::OptionTags::SystemGroup;
+};
+
+/*!
+ * \brief Groups option tags related to the divergence cleaning of the GRFFE
+ * system.
+ */
+struct ConstraintDampingGroup {
+  static std::string name() { return "ConstraintDamping"; }
+  static constexpr Options::String help{
+      "Options related to constraint damping"};
+  using group = ForceFreeGroup;
+};
+
```

Comment:
The force-free current includes a damping term, that acts as an effective resistivity.
It might make sense to add an option for that, too. Sometimes this needs to be adjusted.
`Eta` might be a good name; it has the same dimensions as the Kappa's

Reply:
Thanks for pointing this out!

From your paper I see `Nu` was used and in Alic's paper `SigmaB` was used. I think we can use `Eta` when we describe our equations, but would prefer some descriptive name inside the code.

Reply:
something like `RelaxationParameter`? I cannot think of a good name on this, but I can put this parameter in a separate option group named like `ForceFreeCurrent:`

Reply:
How about `ParallelConductivity`? Physically that parameter is the conductivity parallel to magnetic field lines.

Reply:
Sounds nice :+1: